netbox/dcim.py

```diff
@@ -769,3 +769,135 @@ def get_power_ports(self, **kwargs):
     def get_power_connections(self, **kwargs):
         """Return power connections """
         return self.netbox_con.get('/dcim/power-connections/', **kwargs)
+
+    def create_rear_port(self, name, device_id, type, **kwargs):
+        """Create a new rear port for a device
+
+        :param name: rear port name
+        :param device_id: device id for the device
+        :param type: the type for the rear port (for instance: 8p8c)
+        :param kwargs: optional fields
+        :return: netbox object if successful otherwise exception raised
+        """
+        required_fields = {
+            "name": name,
+            "device": device_id,
+            "type": type
+        }
+        return self.netbox_con.post('/dcim/rear-ports/', required_fields, **kwargs)
+    
+    def get_rear_ports(self, **kwargs):
+        """Return rear ports"""
+        return self.netbox_con.get('/dcim/rear-ports/', **kwargs)
+    
+    def update_rear_port(self, rear_port_id, **kwargs):
+        """Update rear port
+
+        :param rear_port_id: Rear port ID to delete
+        :param kwargs: Items to update
+        :return bool True if successful otherwise raise Exception
+        """
+        return self.netbox_con.patch('/dcim/rear-ports/', rear_port_id, **kwargs)
+    
+    def update_rear_port(self, rear_port_id, **kwargs):
```

Comment:
This is a duplicate of the method above.

Reply:
Yep, you're right. Fixed it.